### DISCUSSION TITLE
Can I add multiple users in AUTH_TYPE=BASIC

### DISCUSSION THREAD

**Discussion by punchh-rakeshj** [provectus/kafka-ui#3823] [Q&A] [answered]
I want to add multiple users e.g. `admin` and `readonly` with respective (full and read-only) permissions. Because RBAC seems only supports OAuth and we are using saml.

**Comment by Haarolean** [comment, marked as answer]
Hi, unfortunately, that's not possible within the current implementation, there's just one pair of credentials.
As for SAML, auth itself is yet not supported (#478).
